Enemy Animations

sportsad opened the pull request:
Implemented animations and edited scripts to trigger animations.

CJSheehan15 committed:
image effects and a lil more to parallax

Standard Assets folder is to keep this time around as it is necessary
for using unity image effects

a lot more of the cave background is implemented in parallax

2 new scripts are custom for implementing image effects

warpaltarpers committed:
Merge pull request #35 from warpaltarpers/envPar&FX

image effects and a lil more to parallax

sportsad committed:
Enemy Animations

Implemented animations and edited scripts to trigger animations.

DaneWini reviewed (changes requested):
Enemy animations aren't there, player cant attack, and the edges of platforms you cant stand on anymore fix what you can please!

sportsad committed:
More enemy animation

sportsad committed:
Added Box Collider to patrol enemy attack

Key frames added to the attack animation to resize and move the box collider for the slash attack of the patrol enemy.
